client/state/timezones/reducer.js

```diff
@@ -27,8 +32,15 @@ export const byContinents = createReducer( {}, {
 	[ TIMEZONES_RECEIVE ]: ( state, actions ) => ( actions.byContinents )
 }, continentsSchema );
 
+export const requesting = createReducer( false, {
+	[ TIMEZONES_REQUEST ]: () => true,
+	[ TIMEZONES_REQUEST_FAILURE ]: () => false,
+	[ TIMEZONES_REQUEST_SUCCESS ]: () => false
+} );
```

Comment:
>this function could be much simpler if it weren't using `createReducer`

This would need a first argument for current state, yeah? Nice concise reducer though.